http/management/src/test/java/io/quarkus/qe/LocalOptionsIT.java

```diff
@@ -0,0 +1,94 @@
+package io.quarkus.qe;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class LocalOptionsIT {
+
+    @QuarkusApplication
+    static final RestService app = new RestService();
+
+    @QuarkusApplication
+    static final RestService custom = new RestService()
+            .withProperty("quarkus.management.port", "9002");
+
+    @QuarkusApplication
+    static final RestService tls = new RestService()
+            .withProperty("quarkus.management.port", "9003")
+            .withProperty("quarkus.management.ssl.certificate.key-store-file", "META-INF/resources/server.keystore")
+            .withProperty("quarkus.management.ssl.certificate.key-store-password", "password");
+
+    @QuarkusApplication
+    static final RestService unmanaged = new RestService()
+            .withProperty("quarkus.management.enabled", "false");
+
+    @QuarkusApplication
+    static final RestService redirected = new RestService()
+            .withProperty("quarkus.management.port", "9004")
+            .withProperty("quarkus.http.root-path", "/api")
+            .withProperty("quarkus.http.non-application-root-path", "/query")
+            .withProperty("quarkus.management.root-path", "management");
+
+    @Test
+    public void greeting() {
+        for (RestService service : Arrays.asList(app, custom, tls, unmanaged)) {
+            Response response = service.given().get("/ping");
+            assertEquals(200, response.statusCode());
+            assertEquals("pong", response.body().asString());
+
+            Response openapi = service.given().get("/q/openapi?format=json"); //openapi should be on the old interface
```

Comment:
This is not exactly a matter of this PR, but is there a clear definition of what has been moved to the management interface? I mean, ideally in the documentation, of course. I can see the items (more or less) in the Quarkus source code, in the Quarkus PR description ([1]), and in the test plan for this feature.

In the docs, there is a mention of `"SmallRye Health Checks, SmallRye Metrics, and Micrometer endpoints"` in a few places (see e.g. [2]), but no definitive list.

Have you been involved in a discussion about this? Maybe I have missed something in the docs or other source?
Because so far it looks to me that, from a user perspective, the list of endpoints is not very obvious and should be IMO better documented.

[1] https://github.com/quarkusio/quarkus/pull/30506
[2] https://quarkus.io/version/main/guides/management-interface-reference#enabling-the-management-interface

Reply:
Ok, but we should make sure to check the docs once the feature goes into product.